### PR TITLE
Disable password reset button for teachers in sections

### PR DESCRIPTION
**Pull request by maddiedierker**
Client changes for [LP-680](https://codedotorg.atlassian.net/browse/LP-680). (Server changes forthcoming.)

### Change 1
If a teacher (A) is in another teacher's (B) section, teacher B can reset teacher A's password via the "Manage Students" tab. This disables the reset password button and displays a tooltip on hover that says 'You cannot reset passwords for teacher accounts in your section.' 

**Examples:**
Picture/word section (treated the same) --
Screen Shot 2019-08-23 at 4 55 04 PM

Email section --
Screen Shot 2019-08-23 at 4 53 55 PM

### Change 2
Hides the "Edit" option from the dropdown menu on the "Manage Students" tab for teachers in the section:
Screen Shot 2019-08-23 at 4 57 12 PM

### CODE REVIEW

File: apps/src/templates/manageStudents/PasswordReset.jsx [hunk 1]
```diff
@@ -72,14 +75,25 @@ class PasswordReset extends Component {
...
-            onClick={this.reset}
-            color={Button.ButtonColor.white}
-            text={i18n.resetPassword()}
-          />
+          <span data-for={tooltipId} data-tip>
+            <Button
+              onClick={this.reset}
+              color={Button.ButtonColor.white}
+              text={i18n.resetPassword()}
+              disabled={this.props.resetDisabled}
```
Comment (davidsbailey) [inline apps/src/templates/manageStudents/PasswordReset.jsx]: this code looks right to me, but the button does not look super disabled in the screenshot for the email section. I would expect the text to be grayed out or something. If it does look disabled to you, maybe you could show a screenshot with a student and teacher row next to each other so the difference is more obvious?

also, nit: can just use `resetDisabled` on this line.

**Review by davidsbailey — APPROVED** [review]
Code looks good overall, but please double check that the resulting UX is what you expect.

**Comment by maddiedierker** [comment]
@davidsbailey good catch! here are better screenshots for a disabled "reset password" button in an email section:

Screen Shot 2019-08-26 at 11 32 15 AM
Screen Shot 2019-08-26 at 11 32 09 AM

**Comment by davidsbailey** [comment]
ok, yeah, it does look more disabled side-by-side. It still doesn't look very disabled to me in isolation, but I think you've done your part here. :-)

**Pull request merged**